main.go

```diff
@@ -254,6 +257,19 @@ func main() {
 		envString("", "GITSYNC_ASKPASS_URL", "GIT_SYNC_ASKPASS_URL", "GIT_ASKPASS_URL"),
 		"a URL to query for git credentials (username=<value> and password=<value>)")
 
+	flGithubAPIURL := pflag.String("github-api-url",
```

Comment:
it's not really a URL is it?  Usually a URL includes "scheme" or "protocol", but later you prepend "https://".  Also - is this allowed to have paths?  E.g. could I say "notgithub.com/mytenant/path" ?

I propose to either make this a URL ("https://host:port/path") or rename it to "host" or "domain".

What do you think?

Reply:
Yeah you're right that 

I think having it as an actual base URL is better, I'll make this change